backend/controllers/cartController.js

```
@@ -79,7 +79,7 @@ const getUserCart = async (req, res) => {
         .json({ success: false, message: "Invalid userId" });
     }
     const userData = await user.findById(userId);
-    let cartData = await userData.cartData;
+    let cartData = (await userData.cartData) || {};
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'cartData' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)